File: Java/BOJ/14916 거스름돈.java
import java.io.*;
import java.util.*;
import java.math.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int n = Integer.parseInt(br.readLine());
        int result = 0;
        int i = 0;
        while(true) {
        	if (n % 5 == 0) {
        		result += n/5;
        		break;
        	}
        	else {
        		n -= 2;
        		result++;
        	}
        	
        	if (n < 0) {
        		break;
        	}
        }
        if (n < 0) {
        	result = -1;
        }
        System.out.println(result);
    }
}

File: Python/BOJ/14916 거스름돈.py
import sys
n = int(sys.stdin.readline())
result, i = 0, 0
while True:
    if n % 5 == 0 :
        result += n//5
        break
    else :
        n -= 2
        result += 1
    
    if n < 0:
        break
if n < 0:
    result = -1

print(result)